service/src/main/java/org/lfenergy/compas/cim/mapping/service/CompasCimMappingService.java

```diff
@@ -65,7 +65,7 @@ public SCL map(List<CimData> cimData, Principal principal) {
      * @return The created SCL Model.
      */
     SCL createBasicSCL(List<CimData> cimData, Principal principal) {
-        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'hh:mm:ss");
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'hh:mm:ssXXX");
```

Comment:
XXX is used for the timezone:
ISO 8601 Time zone

Source:
https://docs.oracle.com/javase/10/docs/api/java/text/SimpleDateFormat.html